#device
string DeviceParent
string DeviceName
string DeviceType
string Architecture
string[] Capabilities
int8 BoardCount
pin[] pins 
